source/MRMesh/MRPointsLoad.cpp

```diff
@@ -192,25 +236,38 @@ tl::expected<MR::PointCloud, std::string> fromPts( std::istream& in )
             return tl::make_unexpected( "Not valid .pts format" );
         cloud.points.push_back( point );
         ++i;
+
+        if ( callback && !( i % 1000 ) )
+        {
+            const float progress = int( in.tellg() - posStart ) / streamSize;
```

Comment:
int( ... ) -> float( ... ), and in other similar places

Reply:
fixed